### DISCUSSION TITLE
n_parts

### DISCUSSION THREAD

**Discussion by rodrigo-pedro** [abetlen/llama-cpp-python#219] [Q&A] [answered]
Hi. I can't find much information on the parameter "n_parts" other than "Number of parts to split the model into". What impact does this setting have? What does partitioning the model achieve?

**Comment by gjmulder** [comment]
https://github.com/ggerganov/llama.cpp/blob/2b2646931bd2a2eb3e21c6f3733cc0e090b2e24b/examples/common.h#L27

From what I can see in the `llama.cpp` source it is normally automatically derived from the model you choose.

**Reply by abetlen** [reply]
@rodrigo-pedro I'm actually not sure, it's from llama.cpp which is why I made it configurable but I've never had to change it. It may make more sense to ask there.

**Reply by abetlen** [reply, marked as answer]
@rodrigo-pedro to answer your question, this is unnused and has been removed in the latest commit https://github.com/ggerganov/llama.cpp/pull/1509
